### DISCUSSION TITLE
Matrix Multiplication

### DISCUSSION THREAD

**Discussion by nonathecoda** [gama-platform/gama#57] [Q&A] [answered]
I'm trying to implement a Kalman Filter in Gama, which is why I need to multiply matrices. However the "*" operator only seems to multiply matrices value by value. Is there any way I can multiply two matrices, like e.g. a 2x3 with a 3x1 matrix?

Thanks!

**Comment by benoitgaudou** [comment, marked as answer]
Dear,

you have the . operator that does the job (not well documented I admit :-)
).
See below :

write matrix([[1],[2]]) . matrix([[1,3]]);

---> 7

write matrix([[1,3]]) . matrix([[1],[2]]);

---->
1;2
3;6

Cheers

Benoit

Le jeu. 1 juin 2023 à 19:52, Antonia Schroff ***@***.***> a
écrit :
